### ISSUE TITLE
Any tutorials?

### ISSUE THREAD

**Issue by Moravia300** [Synthesis-Collective/facefixer#4]
This tool is unlike any modding tool I ever used before, so some kind of tutorial would be nice. And I don't mean for Synthesis itself,  but for this patcher in particular.

**Comment by Heartbreaker714** [comment]
Figured this out and it's great for bulk NPC changes!!

1. Go into "settings" 
2. Under target level select the source mod you want to carry over the changes from
3. Select which changes you want
4. Run it

..Done.

**Comment by Noggog** [comment]
Yeah, I'd like to get some docs on it.  Problem is i didn't write it, nor have i used it (i dont actually play nowadays).   So hoping someone else whos actually put it to heavier use can step up and write out some tutorials for the dials it has, and when/why you would use the patcher to solve XYZ.  If anyone writes anything up, post it here or ping me on the discord, and I'll slide it in!

**Comment by Langmans** [comment]
What I'm thinking is that one should have their NPC visual overhauls at the top of their load order (so as close to skyrim.esm as the base game esm files as possible), and then have facefixer copy appearance to synthesis.esp. That way, records from mods like AI overhaul will be copied into synthesis.esp first, and then the appearance parts will be written over them.

**Comment by hyperion032020** [comment]
This is an easy way to forward all NPC appearance data from whatever plugins you specify. Pick which plugins contain the appearances you desire, place them in whatever order you need, and pick what appearance data you wish to forward (height, eye color, face part, etc). Then run Synthesis. This should help to eliminate the need for NPC appearance patches (at least most of them at any rate).

**Comment by tim-elmer** [comment]
Looking a bit deeper, the "Prioritize by Specified Order" checkbox comes into play when you have multiple mods listed in the target. Similar to your load order, the last (bottom) mod gets applied last, and thus "wins".

Keep in mind that some patches _are_ still necessary. For example, some of JK's location overhauls move NPCs from where AI Overhaul expects them to be, and this won't fix that.

**Comment by kennethultimate02** [comment]
Just want to ask. Does the position of my appearance matter on face fixer? For example, the one appearing on the top is prioritized while below is the least. Note I'm referring the position in facefix not in mo2. I want to make Faithful faces overwritten by the other mods on its top

Files to map to:
  Improved High-Poly NPCs - CC Saints and Seducers.esp
  Improved High-Poly NPCs - Madmen.esp
  Improved High-Poly NPCs - Environs - Abandoned Abodes.esp
  Improved High-Poly NPCs - Environs - Tundra Farmhouse.esp
  Faithful Faces - High Poly Head.esp
